src/storage/txn/mod.rs

```diff
@@ -56,6 +56,54 @@ pub enum ProcessResult {
     },
 }
 
+// TODO: error info lost!
+impl ProcessResult {
```

Comment:
Iff the `ProcessResult` is the result of a successful `acquire_pessimistic_lock`, the `ProcessResult` will be cloned. I think it needn't handle impossible results here.

Reply:
If so, `Error` in `storage/errors.rs` needn't be clonable.

Reply:
If so, `Error` in `storage/errors.rs` needn't be clonable.

Reply:
Addressed